PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                    <locus from="1r" to="62r"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The manuscript can be dated to the fourteenth or fifteenth century based on palaeographic features. For example, the letter መ shows a tendency for loop separations, ሎ demonstrates the
+                                    vowel marker with a linking line, ጵ exhibits the form used before the sixteenth century. Additionally, the word እግዚአብሔር is 
+                                    written as a single word. Overall, the manuscript displays the standard orthography 
+                                    of medieval Ethiopic. </desc>
+                               <seg type="ink">Black, red.</seg>
+                                <date notBefore="1400" notAfter="1500"> </date>
+                               </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                               <item xml:id="e1">
+                                   <locus target="#44v #45"></locus>
+                                   <desc>The name <persName ref="PRS14538AskalaGiyorgis"/> is written in supplications over erasure, 
+                                       possibly close to the time of the production of the main text. </desc>
+                                   <q xml:lang="gez"><seg part="I">ወናዝዝ፡ ሐዘንየ፡ ሊተ፡ ለገብርከ፡<persName ref="PRS14538AskalaGiyorgis">
+                                       አስካለ፡ ጊዮርጊስ፡</persName>
+                                      </seg></q>
+                                </item>
+                               </list>
+                        </additions>                     
+                           
+                        <bindingDesc>
+                            <binding contemporary="unknown" xml:id="binding">
+                                <decoNote xml:id="b1">The boards of the manuscript are missing. The textblock is held together by the sewing.</decoNote> 
+                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                          <origDate notBefore="1400" notAfter="1500" evidence="lettering">
+                                1400–1500 </origDate>.
+                        </origin>
+                        <provenance>No colophon notes are present in the manuscript, but the owner 
+                            <persName ref="PRS14417MarigetaDamseEndalew" role="owner"/>
+                            who lives in<placeName ref="LOC2386DabraT">
```

Comment:
I think, you need to add a space after in, otherwise it will be printed "inDabra Tābor ʾIyasus parish".

Reply:
You need three more commas here: 1) after 'owner', 2) after 'Marigeta', 3) after 'Dabra Tabor'


There are no colophons in the manuscript, but the owner, <persName ref="PRS14417MarigetaDamseEndalew" role="owner"/>, who lives in <placeName ref="LOC2386DabraT">Dabra Tābor ʾIyasus parish</placeName>, confirmed that the manuscript has been handed down in his family for 6 generations.